<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>426</width>
     <height>426</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="led_1">
      <property name="styleSheet">
       <string notr="true">border: 2px solid #8f8f91;
/*
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                            stop: 0 #f6f7fa, stop: 1 #dadbde);
*/
/*background-color:rgb(255, 89, 255);*/
width:80px;
height:80px;

border: 2px solid rgb(170, 255, 0);
border-radius: 40px;
text-align: center;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="led_4">
      <property name="styleSheet">
       <string notr="true">border: 2px solid #8f8f91;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                            stop: 0 #f6f7fa, stop: 1 #dadbde);
width:80px;
height:80px;

border: 0px solid #e4007e;
border-radius: 40px;
text-align: center;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="led_5">
      <property name="styleSheet">
       <string notr="true">border: 2px solid #8f8f91;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                            stop: 0 #f6f7fa, stop: 1 #dadbde);
width:80px;
height:80px;

border: 0px solid #e4007e;
border-radius: 40px;
text-align: center;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="led_7">
      <property name="styleSheet">
       <string notr="true">border: 2px solid #8f8f91;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                            stop: 0 #f6f7fa, stop: 1 #dadbde);
width:80px;
height:80px;

border: 0px solid #e4007e;
border-radius: 40px;
text-align: center;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="led_2">
      <property name="styleSheet">
       <string notr="true">border: 2px solid #8f8f91;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                            stop: 0 #f6f7fa, stop: 1 #dadbde);
width:80px;
height:80px;

border: 0px solid #e4007e;
border-radius: 40px;
text-align: center;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QPushButton" name="led_9">
      <property name="styleSheet">
       <string notr="true">border: 2px solid #8f8f91;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                            stop: 0 #f6f7fa, stop: 1 #dadbde);
width:80px;
height:80px;

border: 0px solid #e4007e;
border-radius: 40px;
text-align: center;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="led_6">
      <property name="styleSheet">
       <string notr="true">border: 2px solid #8f8f91;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                            stop: 0 #f6f7fa, stop: 1 #dadbde);
width:80px;
height:80px;

border: 0px solid #e4007e;
border-radius: 40px;
text-align: center;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="led_8">
      <property name="styleSheet">
       <string notr="true">border: 2px solid #8f8f91;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                            stop: 0 #f6f7fa, stop: 1 #dadbde);
width:80px;
height:80px;

border: 0px solid #e4007e;
border-radius: 40px;
text-align: center;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="led_3">
      <property name="styleSheet">
       <string notr="true">border: 2px solid #8f8f91;
/*
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                            stop: 0 #f6f7fa, stop: 1 #dadbde);
*/
background-color:rgb(255, 89, 255);
width:80px;
height:80px;

border: 2px solid rgb(170, 255, 0);
border-radius: 40px;
text-align: center;</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
